plotly/figure_factory/_dendrogram.py

```diff
@@ -88,20 +89,19 @@ def create_dendrogram(X, orientation="bottom", labels=None,
 
     dendrogram = _Dendrogram(X, orientation, labels, colorscale,
                              distfun=distfun, linkagefun=linkagefun,
-                             hovertext=hovertext)
+                             hovertext=hovertext, color_threshold=color_threshold)
 
-    return graph_objs.Figure(data=dendrogram.data,
-                             layout=dendrogram.layout)
+    return graph_objs.Figure(data=dendrogram.data, layout=dendrogram.layout)
 
 
 class _Dendrogram(object):
     """Refer to FigureFactory.create_dendrogram() for docstring."""
 
     def __init__(self, X, orientation='bottom', labels=None, colorscale=None,
-                 width=np.inf, height=np.inf, xaxis='xaxis', yaxis='yaxis',
+                 width="100%", height="100%", xaxis='xaxis', yaxis='yaxis',
```

Comment:
"100%" isn't a valid for a width/height. This is what's causing your test failure. I think this should probably be `None`, which will let the rendering approaches (`plot`, `iplot`, or `FigureWidget`) decide what size to use.